js/piemenus.js

```diff
@@ -3439,7 +3439,15 @@ const piemenuBlockContext = (block) => {
         });
     };
 
-    wheel.navItems[0].navigateFunction = stackPasting;
+    wheel.navItems[0].navigateFunction = () => {
+        if (
+            "customsample" === block.blocks.blockList[topBlock].name
+        ) {
+            that.activity.errorMsg(_("Reload the widget, import the sample again, and export."));
```

Comment:
This message is a bit terse. Maybe say, "In order to copy a sample, you must..."

Reply:
Okay will do it.